src/GeositeFramework/locales/en.json

```diff
@@ -15,7 +15,7 @@
     "Force maps to show the same extent": "Force maps to show the same extent",
     "Link Maps": "Link Maps",
     "Receive a printable copy of the current map": "Receive a printable copy of the current map",
-    "Export Page": "Export Page",
+    "Create Map": "Create Map",
```

Comment:
They keys should be the same between both files (although it's possible for keys to be missing if we forgot to add a translated version).

https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/prototype/src/GeositeFramework/locales/es.json#L18